style.css

```diff
@@ -40,13 +54,59 @@ p {
   width: 18em;
 }
 
+section {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
 ul {
   list-style-type: none;
-  padding-inline-start: 0;
+  white-space: nowrap;
+  padding-left: 0;
 }
 
 /* Classes */
 
+/* FlexBox and GridLayout */
+
+.d-flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.flex-grow {
+  flex-grow: 1;
+}
+
+.grid_5x1 {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto;
+  grid-template-columns: auto;
+  justify-items: center;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, I suggest to keep stcicking on the design, kindley take into account the background, paddings and margins as per the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=NYF0n2mQq81VU7Uz-0):

  - Your design:
![Your1](https://user-images.githubusercontent.com/97834160/205410396-c9803950-56da-448d-baf6-9688d1e838fd.png)

  - Figma:
![fig1](https://user-images.githubusercontent.com/97834160/205410406-1d879b4f-2a04-4907-93fa-8dba74259214.png)